doit/dependency.py

```diff
@@ -527,23 +588,27 @@ def get_status(self, task, tasks_dict):
                 task.dep_changed = list(task.file_dep)
                 return 'run'
 
+        # check for modified file_dep checker
```

Comment:
I would implement in a different way but I think this way is good because it can also handle the case where each task uses a different Checker (in case it gets implemented in the future).